src/scss/cs/competition.scss

```diff
@@ -0,0 +1,51 @@
+.ioe {
+  &-row {
+    gap: 40px;
+    justify-content: center;
+  }
+
+  &-col {
+    background-color: white;
+  }
+
+  &-custom-width {
+    width: 90%;
+  }
+
+  @media (min-width: 1200px) {
+    &-custom-width {
+      width: 540px;
+    }
+  }
+
+  &-title {
+    color: #120540;
+    font-size: 24px;
+  }
+
+  &-text {
+    color: #1c1c1c;
+    font-size: 19px;
+  }
+
+  &-comment {
+    display: flex;
+    align-items: center;
+
+    &-content {
+      background-color: #fff5fa;
+      color: #120540;
+      font-size: 16px;
+      font-family: "roboto", Roboto, sans-serif;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      padding: 20px 65px 20px 65px;
+      text-align: center;
+
+      @media (max-width: 576px) {
+        width: 90%;
```

Comment:
again, if the guttering and the flex of the items here is right, this should be totally unnecessary

Reply:
Agreed